tracer/build/crank/os.profiles.yml

```diff
@@ -87,26 +87,21 @@ profiles:
         endpoints:
           - "{{ linuxEndpoint }}"
         environmentVariables:
-          COR_PROFILER: "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}"
-          COR_PROFILER_PATH: "{{ linuxProfilerPath }}/monitoring-home-linux/Datadog.Trace.ClrProfiler.Native.so"
           CORECLR_PROFILER: "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}"
-          CORECLR_PROFILER_PATH: "{{ linuxProfilerPath }}/monitoring-home-linux/Datadog.Trace.ClrProfiler.Native.so"
+          CORECLR_PROFILER_PATH: "{{ linuxProfilerPath }}/tracer-home-linux/linux-x64/Datadog.Trace.ClrProfiler.Native.so"
           DD_DOTNET_TRACER_HOME: "{{ linuxProfilerPath }}/tracer-home-linux"
-          DD_DOTNET_PROFILER_HOME: "{{ linuxProfilerPath }}/profiler-home-linux"
+          DD_DOTNET_PROFILER_HOME: "{{ linuxProfilerPath }}/tracer-home-linux/linux-x64"
           DD_AGENT_HOST: "{{ controllerIp }}"
           DD_TRACE_LOGGING_RATE: 6
-          DD_TRACE_DEBUG: 0
           DD_ENV: throughput-adhoc
-          LD_LIBRARY_PATH: "{{ linuxProfilerPath }}/profiler-home-linux"
-          LD_PRELOAD: "{{ linuxProfilerPath }}/profiler-home-linux/Datadog.Linux.ApiWrapper.x64.so"
+          LD_LIBRARY_PATH: "{{ linuxProfilerPath }}/tracer-home-linux/linux-x64"
+          LD_PRELOAD: "{{ linuxProfilerPath }}/tracer-home-linux/linux-x64/Datadog.Linux.ApiWrapper.x64.so"
+          DD_PROFILING_ENABLED: 1
```

Comment:
After a super quick glance, I don't know where those profiles are used. Most changes LGTM but you're adding DD_PROFILING_ENABLED whereas it doesn't seem to be needed for this PR. As I don't understand where it's used, could you enlighten me and tell me what it's going to impact :prettyplease:?

Reply:
I setup an environment design to be launched from DD employees laptops directly, so that you can do performance testing in AWS, but outside of CI. This can be done using the script `./run-adhoc.sh`, which is described in the crank readme. There are two setups configured for this one that runs with the profiler and that runs without it.